pkg/controller/tektoninstallation/tektoninstallation_controller_test.go

```diff
@@ -60,10 +65,102 @@ func TestTektonInstallationController(t *testing.T) {
 				Exists().
 				HasSpec(tektonSub.Spec)
 
+			AssertThatTektonInstallation(t, tektonInstallation.Namespace, tektonInstallation.Name, cl).
+				HasConditions(Unknown())
+		})
+
+	})
+
+	// reconciling on tektonconfig resource watcher
+	t.Run("tektonconfig watcher", func(t *testing.T) {
+
+		t.Run("installed tekton installation", func(t *testing.T) {
+			// given
+			tektonInstallation := NewInstallation()
+			tektonConfig := NewTektonConfig("applied-addons", "installed", "validated-pipeline")
```

Comment:
you could use the constant instead of the "installed" string:
```suggestion
			tektonConfig := NewTektonConfig("applied-addons", config.InstalledStatus, "validated-pipeline")
```

Reply:
@MatousJobanek sure, i will fix this - at one point we were avoiding using consts in our tests because we wanted to make sure that the right values were used (this was a comment i received on a previous PR awhile ago) hence why i avoided using the consts..

Reply:
yes, but that was the usage of our constants if I'm not mistaken. Correct me If I'm wrong.
If you have a reasoning for keeping the string instead of constants, please just write it here.